assets/src/edit-story/components/focusable/navigableGroup.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useRef, useLayoutEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import getDirectionalKeysForNode from '../../utils/getDirectionalKeysForNode';
+import Focusable from './focusable';
+
+function NavigableGroup( { direction, element, className, role, tabindex, children } ) {
+	const ref = useRef();
+	const [ shortcuts, setShortcuts ] = useState( {} );
+
+	useLayoutEffect( () => {
+		const focus = ( node ) => node && node.focus();
+
+		if ( direction === NavigableGroup.DIRECTION_HORIZONTAL ) {
+			const { forward, backward } = getDirectionalKeysForNode( ref.current );
```

Comment:
This makes sense, but to clarify: what does `Tab` do inside the NavigableGroup?

Reply:
Pressing <kbd>tab</kbd> navigates between focusable elements using browser-native implementation. Just like <kbd>enter</kbd> clicks the currently focused element - this is also using browser-native behaviour. I believe that's excellent for a11y.

Reply:
Ok. I think that's good. In some systems <kbd>tab</kbd> jumps between bigger blocks and arrows are used to navigate within the block. But I think for now a simple way is better. This might pop up for e.g. gallery, where there might be unlimited set of things to <kbd>tab</kbd> over, which could make <kbd>tab</kbd> harder to use to navigate to other elements.

Reply:
Looking at the tablist implementation at https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html and the one I built for WordPress core (click on share icon on https://wordpress.org/news/2019/12/wordpress-5-3-1-security-and-maintenance-release/embed/), pressing <kbd>tab</kbd> jumps to the current tab's content, not to the next tab.

Reply:
> Looking at the tablist implementation at https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html and the one I built for WordPress core (click on share icon on https://wordpress.org/news/2019/12/wordpress-5-3-1-security-and-maintenance-release/embed/), pressing tab jumps to the current tab's content, not to the next tab.

Thanks a lot for those links - very helpful!

Those definitely are different ways of implementing this. The inactive tabs don't participate in tab order at all it seems (`tabindex="-1"`) - only the active tab and the tab content. <kbd>tab</kbd> and <kbd>shift</kbd>+<kbd>tab</kbd> then navigate between the active tab and the tab content in the regular way based off document order.

And they have an example of both automatic and manual tab switching. Very interesting approaches - I will read up on those!